### PR TITLE
exposing stats in hermes-console

### PR DESCRIPTION
**Pull request by adamdubiel**
* exposing new stats in hermes-console
* exposing ack-all/ack-leader and tracking settings in console

### CODE REVIEW

**Comment by wojtkiewicz** [comment]
:+1:

File: hermes-console/static/css/main.css [hunk 1]
```diff
@@ -53,3 +53,8 @@ span.helpme {
 a.navbar-brand.console-title {
     color: #2a6496;
 }
+
+div.stats  {
+    /*border-top: 1px solid #bbb;*/
+    /*background: #f8f8f8;*/
+}
```
Comment (dankraw) [inline hermes-console/static/css/main.css]: committed accidentally?

File: hermes-api/src/main/java/pl/allegro/tech/hermes/api/endpoints/QueryEndpoint.java [hunk 2]
```diff
@@ -16,20 +18,21 @@
     @POST
     @Consumes(APPLICATION_JSON)
     @Produces(APPLICATION_JSON)
-    @Path("/group")
+    @Path("/groups")
     List<Group> queryGroup(String query);
```
Comment (dankraw) [inline hermes-api/src/main/java/pl/allegro/tech/hermes/api/endpoints/QueryEndpoint.java]: we could rename these methods by adding "s", we `queryGroups`, `queryTopics`...

File: hermes-console/static/partials/modal/editTopic.html [hunk 3]
```diff
@@ -23,24 +23,41 @@ <h3 class="modal-title" ng-show="operation === 'EDIT'"><small>Edit topic:</small
                     <input class="form-control" required id="topicDescription" name="topicDescription" placeholder="what type of messages you publish?" ng-model="topic.description"/>
                 </div>
             </div>
+            <div class="form-group {{topicForm.topicRetention.$valid ? '' : 'has-error'}}">
+                <label for="topicRetention" class="col-md-3 control-label">Retention time</label>
+                <div class="col-md-9">
+                    <div class="input-group">
+                        <input type="number" min="0" max="7" required class="form-control" id="topicRetention" name="topicRetention" ng-model="topic.retentionTime.duration" placeholder="how much time should we keep them for subscribers?"/>
```
Comment (dankraw) [inline hermes-console/static/partials/modal/editTopic.html]: how much time -> how long ?

File: hermes-console/static/partials/search.html [hunk 4]
```diff
@@ -5,32 +5,62 @@
...
                     <span class="glyphicon glyphicon-chevron-right"></span>
                 </div>
+
+                <h4 class="list-group-item-heading">
+                    {{item.name}}
+                </h4>
+
+                <p class="list-group-item-text" style="margin-top: 1em;">
+                    <span class="" ng-repeat="object in item.data">
+                        <strong>{{object.label}}:</strong> {{object.value}} &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```
Comment (dankraw) [inline hermes-console/static/partials/search.html]: :)

**Comment by dankraw** [comment]
:+1:

File: hermes-console/static/partials/search.html (hunk 4 above)
Comment (adamdubiel) [inline hermes-console/static/partials/search.html]: simplicity ;)

**Comment by ingwarsw** [comment]
:+1:

**Pull request merged**